.github/workflows/ios-build-test-fabric.yml

```diff
@@ -59,4 +59,4 @@ jobs:
 
       - name: Build app
         working-directory: ${{ env.WORKING_DIRECTORY }}
-        run: yarn ios
+        run: npx react-native run-ios --no-packager --simulator="iPhone 15"
```

Comment:
* `--no-packager` - because something changed & `yarn run` / `react-native` cli (not sure, I didn't see need to investigate it) attempts to spawn new terminal session in **new window**, but there is no graphical session on CI -> workflow crashes.
* `--simulator` - because something changed and when no simulator is specified it tries to launch *first available*, so happens that the first one is Apple TV simulator -> workflow crashes.

Reply:
> [...] but there is no graphical session on CI -> workflow crashes.

I read it once again & I started wondering how emulators / simulators work ;D Turns out I do not understand the issue thoroughly. Nevertheless this is the solution...